staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -127,13 +127,19 @@ func NewReflector(lw ListerWatcher, expectedType interface{}, store Store, resyn
 
 // NewNamedReflector same as NewReflector, but with a specified name for logging
 func NewNamedReflector(name string, lw ListerWatcher, expectedType interface{}, store Store, resyncPeriod time.Duration) *Reflector {
+	realClock := &clock.RealClock{}
 	r := &Reflector{
 		name:          name,
 		listerWatcher: lw,
 		store:         store,
-		period:        time.Second,
-		resyncPeriod:  resyncPeriod,
-		clock:         &clock.RealClock{},
+		// this will result in a 0.1sec, 0.2sec, 0.4sec, 0.8sec, 1.6sec, 3.2sec, 6.4sec, 12.8sec, 25.6sec,
+		// 30sec, 30sec, ... backoff. With such backoff, we have ~95% traffic reduction (0.05 QPS) after 1min,
+		// and ~98% traffic reduction (0.02 QPS) after ~1.5min.
```

Comment:
I guess I don't fully understand this math - after 1 min, we will be sending 1 request per 30s, so ~0.03QPS.

The argument with traffic reduction was more comparing to the current state (where we're sending request every 1s), so I think this % reduction doesn't make much sense in the comment file.

Reply:
Seems like I made a bad estimation. I think the jitter would even make the exact math more complicated here. For comments, I will just make it simple - 
we used to make the call every 1sec, our goal is to make a 98% traffic reduction (~0.02) QPS by stop at 30sec backoff with 1.0 jitter (random between 30sec and 60sec.)